From 4431ad34734632c439881a30ca90617a1fd1dde4 Mon Sep 17 00:00:00 2001
From: Dmitry Rozhkov <dmitry.rozhkov@linux.intel.com>
Date: Tue, 23 Feb 2016 14:50:37 +0200
Subject: [PATCH] Clean up tar options: drop -a for the extract mode.

The -a option is not needed for the extract mode since tar
auto-detects compression type from file format.

Upstream-Status: Accepted

Signed-off-by: Dmitry Rozhkov <dmitry.rozhkov@linux.intel.com>
---
 src/pack.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/pack.c b/src/pack.c
index 5bb99b8..e5203d0 100644
--- a/src/pack.c
+++ b/src/pack.c
@@ -114,7 +114,7 @@ static void explode_pack_stage(int version, char *module)
 		 * time on the client...
 		 */
 		string_or_die(&tar, "tar --directory=%s/%s/%i/staged --warning=no-timestamp "
-				TAR_PERM_ATTR_ARGS " -axf %s", packstage_dir, module, version, path);
+				TAR_PERM_ATTR_ARGS " -xf %s", packstage_dir, module, version, path);
 		ret = system(tar);
 		if (!ret) {
 			unlink(path);
-- 
2.5.0

